src/chrome/content/rules/ESEA.net.xml

```diff
@@ -10,6 +10,8 @@
 	<target host="mdl.esea.net" />
 	<target host="news.esea.net" />
 	<target host="play.esea.net" />
+	<target host="static.esea.net" />
+		<test url="http://static.esea.net/" />
```

Comment:
This test url returns a 403 status code. Please change to something that response with 200.

Reply:
@J0WI The newest commit on this PR now contains a test URL which should return a 200 status code. From what I understand, the `static.esea.net` is probably exclusively used for the hosting of .cfg files.